server/boot/adminUser.js

```diff
@@ -5,21 +5,21 @@ const { app } = require("../config/config");
 const { make } = require('../services/jwtService');
 const { generateRandomValueBase64 } = require('../utils')
 
-module.exports = App => {
-  const { user } = App.models;
-  user.create(
-    {
-      name: app.admin.name,
-      email: app.admin.email,
-      password: generateRandomValueBase64(20),
-      role: "admin"
-    }, async (err, user) => {
-      if (err) {
-        throw new Error(`Cannot create admin`, err);
-      }
+// module.exports = App => {
```

Comment:
Please, delete this file. This file is no longer needed since we moved from memory to MySQL, so there's no need to seed a new admin user when the app boots.